package/MDAnalysis/core/selection.py

```diff
@@ -794,7 +794,73 @@ def _sel_with_icodes(self, vals, icodes):
         return mask
 
 
+class BoolSelection(Selection):
+    """Range selection for boolean values"""
```

Comment:
How is this a range selection? Do you mean in the sense it selects all for which the selection keyword is true?

Reply:
Sorry, that was a mistake -- I copied from FloatRangeSelection and forgot to edit it properly.